tests/everflow/everflow_test_utilities.py

```diff
@@ -68,11 +70,10 @@ def setup_info(duthost, testbed):
     #
     # TODO: Add a namedtuple to make the groupings more explicit
     def get_port_info(in_port_list, out_port_list, out_port_ptf_id_list, out_port_lag_name):
-        ptf_port_id = ""
         out_port_exclude_list = []
-
         for port in in_port_list:
             if port not in out_port_list and port not in out_port_exclude_list and len(out_port_list) < 4:
+                ptf_port_id = ""
```

Comment:
`ptf_port_id = str(mg_facts["minigraph_port_indices"][port])`?

Reply:
Thank you! 🤦